problems/1.types+operators.js

```diff
@@ -2,19 +2,21 @@
 * @param {any} val - The input value
 * @returns {boolean} - Is val a number
 */
-function isNumber() {
-
+function isNumber(val) {
+  let value = number 
+  return (value === number )? true : false
```

Comment:
Your parameter is called val. You need to check the typeof val and see if that is equal to "number" (what typeof some number will evaluate too)